src/System.Private.ServiceModel/tests/Common/Infrastructure/src/testproperties.props

```diff
@@ -43,13 +43,13 @@
         <Value>$(BridgeRemoteEnabled)</Value>
         <DefaultValue>false</DefaultValue>
       </TestProperty>
-      <TestProperty Include="BridgeCertificateAuthority">
-        <Value>$(BridgeCertificateAuthority)</Value>
-        <DefaultValue>WcfBridge_RootCA.cer</DefaultValue>
+      <TestProperty Include="BridgeCertificatePassword">
+        <Value>$(BridgeCertificatePassword)</Value>
+        <DefaultValue>test</DefaultValue>
       </TestProperty>
-      <TestProperty Include="BridgeHttpsCertificate">
-        <Value>$(BridgeHttpsCertificate)</Value>
-        <DefaultValue>WcfBridge_localhost.pfx</DefaultValue>
+      <TestProperty Include="BridgeCertificateValidityPeriod">
+        <Value>$(BridgeCertificateValidityPeriod)</Value>
+        <DefaultValue>1.00:00:00</DefaultValue>
```

Comment:
Elsewhere I commented the cert generator should have a configurable validity period.  Here it is!  Can the cert generator use this config property rather than a constant?  Is it intended for the same purpose?